devdocs/process.md

```diff
@@ -68,8 +66,9 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
    to your branch
         
        ```
-       git pull upstream 
-       git merge master
+       git fetch upstream 
+       git checkout master
+       git merge upstream/master
```

Comment:
I see, I was referencing from [GitHub's help](https://help.github.com/articles/syncing-a-fork/), but our situation is slightly different.

According to Git's manual, `pull` "incorporates changes from a remote repository into the current branch", so the merging might be done against the current branch rather than local `master`?

It appears to me the original two commands would fetch `upstream` and merge any changes in the remote default branch into the local current branch (possibly `Issue___`), and then merge local `master` into the local current branch. Or perhaps I'm understanding something wrong and `pull` does update local `master` regardless of the local current branch?

If I am understanding it right, would this sequence work better?
```
git checkout master
git pull upstream master
```
```
git checkout Issue{issue number}
git merge master
```
*i.e.* ensure local `master` is in sync with `upstream/master`, then switch to any branches that requires syncing and sync them against local `master`.  
Altering the description with pre-conditions might work too.